airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/typing_deduping/JdbcSqlGenerator.kt

```diff
@@ -265,13 +265,20 @@ abstract class JdbcSqlGenerator(protected val namingTransformer: NamingConventio
                     .toList()
             )
         }
+
+        val dropTableStep =
+            DSL.dropTableIfExists(DSL.quotedName(stream.id.finalNamespace, finalTableIdentifier))
+        val dropTableSql =
+            if (cascadeDrop) {
+                dropTableStep.cascade().getSQL(ParamType.INLINED)
+            } else {
+                dropTableStep.getSQL(ParamType.INLINED)
+            }
+
```

Comment:
nitpick:
```suggestion

        val dropTableStep =
            DSL.dropTableIfExists(DSL.quotedName(stream.id.finalNamespace, finalTableIdentifier))
        if (cascadeDrop) {
            dropTableStep.cascade()
        } 
        val dropTableSql = dropTableStep.getSQL(ParamType.INLINED)

```

and the `dropTableSql` param is probably unnecessary

Reply:
... I did not realize jooq objects actually mutate themselves, had assumed they returned a copy. neat!